### ISSUE TITLE
[BUG] People Tab settings should not be browser-dependant

### ISSUE THREAD

**Issue by diyoyo** [immich-app/immich#6012]
### The bug

Hi,
whenever you connect to Immich from a different browser, you have to reedit the settings if you want to show the People's tab on the left menu.
I believe this should be saved in the database rather than in the local browser settings.

### The OS that Immich Server is running on

Debian (Raspberry OS Full, latest) latest

### Version of Immich Server

v1.91.4

### Version of Immich Mobile App

v1.91.1

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
unchanged
```

### Your .env content

```Shell
unchanged
```

### Reproduction steps

```bash
1. Go to your user settings
2. Enable "People's tab"
3. Open a different browser
4. Log in to your user account
5. People tab is missing, you have to re-enable it in the settings
```

**Comment by jakub-antos** [comment]
Up^

**Comment by amadeous** [comment]
I agree with @diyoyo, this choice should be user-dependent and not depend on the browser.

Do we have a consensus on this issue so that a fix can be made?

**Comment by iamamansharma** [comment]
can you assign this issue to me?

**Comment by amadeous** [comment]
At this point, I would say that you can just work on this issue and submit a PR.. If nobody acted to assign it to you, you may be free to work on it without interferences.. 😆

**Comment by jrasm91** [comment]
Fixed in #12099
